docs/csharp/language-reference/operators/snippets/shared/TypeTestingAndConversionOperators.cs

```diff
@@ -77,7 +77,7 @@ private static void IsOperatorDeclarationPattern()
     private static void AsOperator()
     {
         // <SnippetAsOperator>
-        IEnumerable<int> numbers = [10, 20, 30];
+        IEnumerable<int> numbers = {10, 20, 30};
```

Comment:
Need to have a `new` expression here:

```suggestion
        IEnumerable<int> numbers = new List<int>(){10, 20, 30};
```